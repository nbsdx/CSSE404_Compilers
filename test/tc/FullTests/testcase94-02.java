

class kfupm{
	public static void main (String [] args){
		
		int a1 = 1;
		int a2 = 2;
		
		boolean a3 = a1 + a2;
		
		
	}
}

class kaust extends kfupm{
	int i;
	int j;
	int k;
	
	public boolean happy(){
		int csse = 0;
		return true;
	}
}