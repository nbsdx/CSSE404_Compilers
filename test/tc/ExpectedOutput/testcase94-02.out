Cannot assign type int to variable a3 of type boolean
